docs/user/metrics/timing_distribution.md

```diff
@@ -7,6 +7,10 @@ To measure the distribution of single timespans, see [Timespans](timespan.md). T
 Timing distributions are recorded in a histogram where the buckets have an exponential distribution, specifically with 8 buckets for every power of 2.
 This makes them suitable for measuring timings on a number of time scales without any configuration.
 
+The effect of Glean's "upload enabled" flag occurs when a timing is stopped. 
```

Comment:
I would simplify that a bit. Calling it "flag" and such seems too wordy.

What about:

```
Timings always span the full length between `start` and `stopAndAccumulate`.
If the Glean upload is disabled when calling `start`, the timer is still started.
If the Glean upload is disabled at the time `stopAndAccumulate` is called, nothing is recorded.
```

Maybe the sentence about `start` could even be left out.